src/features/shareModal/components/ShareModal.tsx

```diff
@@ -59,41 +69,19 @@ export const ShareModal = ({ showModal, setShowModal, data }: ShareModalProps) =
         <p>{data.source}</p>
         <h3>{data.title}</h3>
         <div className="shareLink">
-          <input type="text" className="link" value={data.link} disabled />
-          <button className="copy" onClick={copyLink}>
+          <input type="text" value={data.link} disabled />
+          <button onClick={copyLink}>
             <VscCopy />
           </button>
         </div>
       </div>
 
       <div className="shareOptions">
-        <FacebookShareButton url={data.link}>
-          <FaFacebookSquare size={32} />
-        </FacebookShareButton>
-
-        <TwitterShareButton url={data.link}>
-          <FaTwitterSquare size={32} />
-        </TwitterShareButton>
-
-        <EmailShareButton url={data.link}>
-          <FaMailBulk size={32} />
-        </EmailShareButton>
-
-        <WhatsappShareButton url={data.link}>
-          <FaWhatsappSquare size={32} />
-        </WhatsappShareButton>
-
-        <TelegramShareButton url={data.link}>
-          <FaTelegram size={32} />
-        </TelegramShareButton>
-
-        <RedditShareButton url={data.link}>
-          <FaRedditSquare size={32} />
-        </RedditShareButton>
-
-        <LinkedinShareButton url={data.link}>
-          <FaLinkedinIn size={32} />
-        </LinkedinShareButton>
+        {sharingButtons.map(({ component: ShareButton, icon: Icon, title: Title }, index) => (
+          <ShareButton key={index} url={data.link}>
```

Comment:
You forgot to pass the `title={data.title}` to the ShareButton

Reply:
Hey @medyo !

I'm confused a bit, which title do you want me to pass to the button, is it the title of the social media platform or title of the article/news/post?

Reply:
@idebenone I meant the article's title, so that it appears on social medias like:
<img width="601" alt="image" src="https://github.com/medyo/hackertab.dev/assets/2389584/e829d5f7-21ee-498a-b48b-872ff8a0f931">

Reply:
Yeah I got your point,
But I dont think its possible, given then I'm using a library to share the url and as we know most of the websites rely on Open Graph meta tags, they dont accept additional parameters in the offsite share url.
But in X(Twitter) it looks like the above image even without supplying the title prop to the button.

Reply:
@idebenone I'm referring to the title that's included in the prompt, not the open graph metas. Please give this code a shot.
```
<ShareButton key={index} url={data.link} title={data.title}>
  <Icon size={32} title={Title} />
</ShareButton>
```
<img width="605" alt="image" src="https://github.com/medyo/hackertab.dev/assets/2389584/17a840e2-9f42-479c-9b80-f043ec7082fb">

Reply:
@medyo 
Done 👍